deploy/olm-catalog/opendatahub/1.1.2/manifests/opendatahub-operator.v1.1.2.clusterserviceversion.yaml

```diff
@@ -0,0 +1,359 @@
+    createdAt: '2021-01-31T00:00:00Z'
+    containerImage: quay.io/opendatahub/opendatahub-operator:v1.1.2
+    certified: 'False'
+  name: opendatahub-operator.v1.1.2
+  namespace: placeholder
+spec:
+  apiservicedefinitions: {}
+  customresourcedefinitions:
+    owned:
+      - description: KfDef is the Schema for the applications API
+        kind: KfDef
+        name: kfdefs.kfdef.apps.kubeflow.org
+        version: v1
+        displayName: Open Data Hub
+        group: kfdef.apps.kubeflow.org
+  description: |
+    The Open Data Hub is a machine-learning-as-a-service platform built on Red Hat's Kubernetes-based OpenShift® Container Platform. Open Data Hub integrates multiple AI/ML open source components into one operator that can easily be downloaded and installed by OpenShift users.
+
+    Open Data Hub operator allows users to install and manage Open Data Hub and Kubeflow components. Users can mix and match tools from each project to fulfill the needs of their use case. Each of the
+    projects share some components, but can be mostly seen as an extension of each other to provide a complete solution for both novice and skilled enterprise users.
+
+    ### Components
+    * JupyterHub v0.3.5 - Open source multi-user JupyterLab notebook platform w/ GPU support and **NEW** High Availability.
+    * Trino v355 - Distributed analytics SQL database that supports multiple data sources
+    * Hue v4.8.0 - A service that provides data exploration on Hive and S3 buckets.
+    * Spark Thrift Server - A service that allows JDBC clients run Spark SQL queries.
+    * Open Data Hub Dashboard v1.0  - A web dashboard that displays installed Open Data Hub components with easy access to component UIs and documentation
+    * Elyra v2.2.4 - JupyterLab notebooks with support for AI workflows
+    * Ceph Nano v0.7 - Minimal Object Storage provided by Ceph for development use
+    * Apache Spark v2.4.5 - Unified analytics engine for large-scale data processing
+    * Prometheus v2.16.0 - Monitoring and alerting tool
+    * Grafana v7.1.1 - Data visualization and monitoring
+    * Airflow v1.10.11 - Workflow management
+    * Seldon v1.2.0 - Open source platform for deploying machine learning models
+    * Argo v2.12.5 - Container-native Workflow Engine
+    * Apache Superset v1.3.2 - **NEW** Open source application for data exploration and visualization
+    * Apache Kafka v2.8.0 - The open source stream processing platform
+    * OpenShift Pipelines v1.3.1 - Cloud-native CI/CD on OpenShift
+    To install one or multiple of these components use the default KfDef provided with the operator.
+
+    ### Kubeflow Components
+    ODH 1.1.2 supports Kubeflow v1.3.0 and some components such as KF Serving and KFP on Tekton from master branch. To install Kubeflow v1.3.0 components please use this example [KfDef](https://raw.githubusercontent.com/opendatahub-io/manifests/v1.3-openshift/distributions/kfdef/kfctl_openshift_v1.3.0_odh.yaml).
+
```

Comment:
We need to change this to say Kubeflow 1.4 and point to the new 1.4 example kfdef

Reply:
Yes. Which example kfdef?

Reply:
This one https://github.com/opendatahub-io/manifests/blob/v1.4.0-rc.2-openshift/openshift/kfdef/kfctl_openshift_v1.4.0_odh.yaml

Reply:
Updated